### DISCUSSION TITLE
how to remove

### DISCUSSION THREAD

**Discussion by rXelelo** [lutris/lutris#5584] [Q&A] [answered]
how to remove Submitted contributions

**Comment by strycore** [comment]
wat

**Reply by rXelelo** [reply]
> wat

![зображення](https://github.com/user-attachments/assets/c815f29f-456a-44b1-93eb-7a40a0097e47)

**Reply by strycore** [reply, marked as answer]
Go back to the installer and click "Delete draft"

**Reply by strycore** [reply]
Also, if this is a game submission and not an installer you just have to wait for a mod to accept or reject it

**Reply by rXelelo** [reply]
> Also, if this is a game submission and not an installer you just have to wait for a mod to accept or reject it

but how many i need to wait i submit new install of minecraft for 7 day

**Reply by strycore** [reply]
You wait until a mod approves / rejects. We do that whenever we feel like doing it and you have no control when this happens. I can be way more than a week sometime.
